examples/federated-learning/tff/nvflare/README.md

```diff
@@ -0,0 +1,297 @@
+   export NVFLARE_WORKSPACE_BUCKET_NAME
+   envsubst < ../examples/federated-learning/tff/nvflare/templates/pv-workspace.yaml.tpl > ${ACM_CONFIG_SYNC_PATH}/base-storage/pv-workspace.yaml
+   ```
+
+1. Regenerate the `kustomization.yaml` file with all the necessary informations:
+
+   ```bash
+   NVFLARE_EXAMPLE_WORKLOADS_KUBERNETES_NAMESPACE="$(terraform output -raw nvflare_namespace)"
+   export NVFLARE_EXAMPLE_WORKLOADS_KUBERNETES_NAMESPACE
+   envsubst < ../examples/federated-learning/tff/nvflare/templates/pv-workspace.yaml.tpl > ${ACM_CONFIG_SYNC_PATH}/base-storage/pv-workspace.yaml
+   ```
+
+### Create NVFlare folder structure
+
+Now that the persistent volume is created, you can start creating the folder structure and upload it to Cloud Storage.
+
+1. Use the NVFlare image previously built on your workstation:
+
+   ```bash
+   docker run -it ${NVFLARE_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID_WITH_TAG} bash
+   ```
+
+1. Run the provisioning tool:
+
+   ```bash
+   cd ${HOME}
+   nvflare provision
+   ```
+
+   You have to choose the non-ha (non high-availability) choice, as ha (high-availability) is not yet supported on Kubernetes. It will create a `project.yml` file that you can customize for your needs.
+
+1. Re-run the provisioning tool inside the container:
+
+   ```bash
+   cd ${HOME}
+   nvflare provision
+   cd ${HOME}/workspace/example_project/prod_00/
+   ```
+
+   The different folders generated represent the infrastructure you will deploy on the reference architecture:
+
+   - `server1` is the server that will aggregate all the results from the computation
+   - `site-1` and `site-2` are the clients that will be connected to the server
+   - `admin@nvidia.com` is the administration client to start and list jobs
+
+### Clone the repository
+
+You will need to clone the NVFlare repository that contains the job you will run on the reference architecture:
+
+1. Download the NVFlare repository
+
+   ```bash
+   cd ${HOME}
+   wget -qO- https://github.com/NVIDIA/NVFlare/archive/refs/tags/2.4.2.tar.gz | tar xvz
+   ```
+
+   In the newly create folder, there is an `examples` folder that contains lots of demo you can use to test NVFlare. The TensorFlow demo is in the `examples/hello-world/hello-tf2`. The job will need to be in a special folder inside the `admin@nvidia.com` called the `transfer` folder to be deployed on each client participating in the computation.
+
+1. Copy this folder in the `transfer` folder of the `admin@nvidia.com` folder:
+
+   ```bash
+   cd ${HOME}/workspace/example_project/prod_00/admin@nvidia.com/transfer
+   cp -R ${HOME}/NVFlare/examples/hello-world/hello-tf2 .
+   ```
+
+1. Copy the whole workspace folder in Cloud Storage.
+
+   The pods will have access to the infrastructure to run jobs:
+
+   ```bash
+   gcloud storage -m cp -r ${HOME}/workspace gs://${NVFLARE_WORKSPACE_BUCKET_NAME}
+   ```
+
+### Deploy the infrastructure
```

Comment:
At this point, we're still in the container, so this section will not work, unless we exit from the container before running other commands.

Reply:
Done

Reply:
This is still applicable, right? Because you're still inside the nvflare container.

Reply:
Example: is the `gcloud` command available?

Reply:
It says line 137 that you can exit the container. We don't need it anymore